packages/offix-client/integration_test/test/offline.test.js

```diff
@@ -51,13 +63,15 @@ describe('Offline mutations', function () {
     author: "new"
   };
 
-  const optimisticOptions = {
-    mutation: ADD_TASK,
-    operationType: CacheOperation.ADD,
-    returnType: "Task",
-    variables: newTask,
-    updateQuery: GET_TASKS
-  };
+  const newOptimisticResponse = () => {
```

Comment:
questioning the need to wrap this stuff up in a function. I think we could keep that the way it was.

Reply:
The problem has to do with the `CREATE_TASK` mutation, if I don't use a function the `optimisticOptions` will be created with `mutation: undefined` because `CREATE_TASK` is assigned in the line `92`, the alternative to the function are:
* forcing the `optimisticOptions` to hold the pointer to the `CREATE_TASK` variable
* assign the `optimisticOptions` after assigns the `CREATE_TASK`
In any case, I would like to refactor again the tests and remove queries and mutations form global variables, the less global variables we use the better it is, but I want to do it in a sperate PR